cccore/Program.cs

```diff
@@ -537,6 +538,24 @@ public static List<string> Run(HttpClient httpClient, string creditcoinUrl, stri
                             Debug.Assert(msg != null || msg == null && link != null);
                             if (msg == null)
                                 return null;
+                            if (command[0].Equals("RegisterAddress", StringComparison.InvariantCultureIgnoreCase))
+                            {
+                                var blockchain = command[1];
+                                var addr = command[2].ToLowerInvariant();
+                                var network = command[3];
+                                var addressHash = (blockchain + addr + network).ToByteArray().ToSha512().ToHexString();
+                                var stateAddress = RpcHelper.creditCoinNamespace + RpcHelper.addressPrefix;
+                                stateAddress += addressHash.Substring(addressHash.Length - 60);
```

Comment:
In [show address](https://github.com/gluwa/Creditcoin-Shared/blob/416436389be43d72ab84f019e912f8b0c027dc62/cccore/Program.cs#L261) the address is fetched with a filter op. Why does this differ from the other function? Yours looks faster on paper, does the lookup rely on the filter op being slow (i.e. waiting until the txn is included in the chain head?